charts/flyte-binary/templates/configmap.yaml

```diff
@@ -68,6 +70,10 @@ data:
       k8s-array:
         logs:
           config: {{- include "flyte-binary.configuration.logging.plugins" . | nindent 12 }}
+      agent-service:
+        defaultGrpcEndpoint: flyteagent.flyte.svc.cluster.local:8000
+        supportedTaskTypes:
+        - bigquery_query_job_task
```

Comment:
Making this configurable through values files can help testing new plugins easily and similarly L62